pubspec.lock

```diff
@@ -518,6 +518,13 @@ packages:
       url: "https://pub.dartlang.org"
     source: hosted
     version: "0.17.0"
+  inview_notifier_list:
+    dependency: "direct main"
+    description:
+      name: inview_notifier_list
```

Comment:
- [We removed `inview_notifier_list` in this PR.](https://github.com/PalisadoesFoundation/talawa/pull/1105/commits/6e55086baa4ea44519f0f39be45612839d666e65). Why are you proposing to restore it?
- [We did further work on doing this here.](https://github.com/PalisadoesFoundation/talawa/pull/1099)
- [We are also trying to remove its duplicated functionality.](https://github.com/PalisadoesFoundation/talawa/issues/1093)

Please find a way to do your testing with the `visibility_detector` package.

Reply:
No I'm not proposing to restore it by any means. I don't know how it came in the PR.

Reply:
Removed that lines now.